pkg/controller/federatedtypeconfig/controller.go

```diff
@@ -145,6 +145,15 @@ func (c *Controller) reconcile(qualifiedName util.QualifiedName) util.Reconcilia
 	limitedScope := c.controllerConfig.TargetNamespace != metav1.NamespaceAll
 	if limitedScope && syncEnabled && !typeConfig.GetNamespaced() {
 		glog.Infof("Skipping start of sync & status controller for cluster-scoped resource %q.  It is not required for a namespaced federation control plane.", typeConfig.GetTemplate().Kind)
+
+		typeConfig.Status.ObservedGeneration = typeConfig.Generation
+
+		_, err = c.client.FederatedTypeConfigs(typeConfig.Namespace).UpdateStatus(typeConfig)
+		if err != nil {
+
+			runtime.HandleError(fmt.Errorf("Could not update ObservedGeneration of the CRD: %q: %v", key, err))
```

Comment:
This block should return `util.StatusError` to ensure the event is processed again in a timely fashion.

Reply:
Ack. I will make the necessary changes.